### PR TITLE
Inline wiki table formatting broken

### PR DESCRIPTION
**Pull request by mardelvalle**
This pull request is related to the following issue: https://github.com/publiclab/plots2/issues/3757#issuecomment-441613619

### CODE REVIEW

File: app/views/notes/_comment.html.erb [hunk 1]
```diff
@@ -126,8 +126,8 @@
     <% end %>
 
     <% if comment.body.include?('?') %>
-    <p class="alert alert-info">Is this a question? <a href="/questions/new?title=<%= comment.body %>">Click here</a> to post it to the <a href="/questions">Questions page</a>.
-    </p>
+      <p class="alert alert-info">Is this a question? <a href="/questions/new?title=<%= comment.body %>">Click here</a> to post it to the <a href="/questions">Questions page</a>.
```
Comment (oorjitchowdhary) [inline app/views/notes/_comment.html.erb]: @mardelvalle You've added spaces in Line 129 and 130.. Pls remove them in order to pass the Travis CI...

Comment (mardelvalle) [reply app/views/notes/_comment.html.erb]: Huh... I thought I had removed that...I will try to remove that again.

Comment (mardelvalle) [reply app/views/notes/_comment.html.erb]: Actually I just checked and it is not appearing in the files changed..

Comment (oorjitchowdhary) [reply app/views/notes/_comment.html.erb]: 😕 I can still see it at https://github.com/publiclab/plots2/blob/24fa038956eaeb79a14d88f63a9efc6333f7a431/app/views/notes/_comment.html.erb

**Comment by jywarren** [comment]
Hmm, i think this may be an unrelated intermittent error we're trying to solve. I restarted the build which you can do yourself by closing and reopening the PR -- let's see if it passes this time!

**Comment by jywarren** [comment]
OK! I merged in a possible fix for the unrelated error and rebased this on top of it. Let's see how it does this time! Sorry for the trouble, it's not related to your code. Thanks for your patience!!!

**Comment by jywarren** [comment]
Awesome! This is done. Great work!!

**Pull request merged**